Resources/Prototypes/Entities/Clothing/Back/backpacks.yml

```
@@ -201,3 +201,14 @@
     equippedPrefix: holding
   - type: Storage
     capacity: 9999
+
+- type: entity
+  parent: ClothingBackpackClown
+  id: ClothingBackpackCluwne
+  name: cursed giggles von honkerton
+  suffix: Unremoveable
```

The way we normally put suffix for mappers is 
```suggestion
  suffix: [Unremoveable]
```
same for the rest

---

it adds the [], you cannot add them currently otherwise suffix would be [[unremoveable]]